src/github.com/couchbase/sync_gateway/rest/server_context.go

```diff
@@ -724,12 +724,22 @@ func (sc *ServerContext) _getOrAddDatabaseFromConfig(config *DbConfig, useExisti
 		revCacheSize = db.KDefaultRevisionCacheCapacity
 	}
 
+	unsupportedOptions := &db.UnsupportedOptions{}
+	if config.Unsupported != nil {
+		if config.Unsupported.UserViews != nil {
+			if config.Unsupported.UserViews.Enabled != nil {
+				unsupportedOptions.EnableUserViews = *config.Unsupported.UserViews.Enabled
+			}
+		}
+	}
+
```

Comment:
I'd coded it with future enhancements to config.Unsupported in mind (where each child of Unsupported would manage it's own config section, and could have multiple items to manage in each), but if you think that's premature optimization/code style I don't mind switching.